### ISSUE TITLE
Scheduling

### ISSUE THREAD

**Issue by GeorgeHahn** [GeorgeHahn/CS431_Project#8]
@adamr5000 @somsai002 

Would you guys like to get together during normal lecture hours on Monday to do some development?

I have a Physics common until 5:45, but I'll be free (albeit fairly hungry) afterwards.

**Comment by adamr5000** [comment]
I'd be fine with meeting then, or if you want to grab dinner first that'd be fine too. I have a 4:00 class but other than that that time frame is pretty much clear.

**Comment by GeorgeHahn** [comment]
Works for me! I'll grab something quick after my test; let's plan on meeting up at 6?

Could you text me again? I switched phones and lost your number along the way. Thanks!

**Comment by somsai002** [comment]
I can do that I'm free from 4 tmmr

**Comment by GeorgeHahn** [comment]
Excellent! See you guys then.

**Comment by somsai002** [comment]
Guys I can't meet up. My earlier class got cancelled too. So Im not going
to come to college tday. Would u guys be able to meet up tmmr. Or sometime
in the week.

**Comment by adamr5000** [comment]
My Tuesdays are actually pretty much completely free, as it turns out. Just have a night class at 6:00. I'm pretty much fine with whatever.

**Comment by GeorgeHahn** [comment]
I can't do this Tuesday, but I may be able to do next.



—

**Comment by somsai002** [comment]
Okay

**Comment by GeorgeHahn** [comment]
To follow up on our discussion in class today, the next meeting will be between 1:00 and 2:30pm on Thursday

**Comment by adamr5000** [comment]
Any ideas for where we should meet today, at least initially?

**Comment by GeorgeHahn** [comment]
I wonder if we could find an open room on the third floor of the CKB

**Comment by adamr5000** [comment]
We could meet outside CKB at 1:00, in that case.

In unrelated news, attached is what I've got so far for the ER diagram, whipped it up relatively quick in Dia. Definitely still needs work, of course; most of the lines aren't even labeled. I'm thinking I'm going to have to use a different program - Dia's nice, but it doesn't have double lines or even underlining text, both of which seem somewhat crucial to the model as he's presented it. Honestly, I would've downloaded a different program, but the Internet on my desktop mysteriously stopped working and I spent about 30 minutes trying to get it fixed before giving up and making something with the program I had. Laptop's good, at least.

![project_erdiagram](https://cloud.githubusercontent.com/assets/11198351/10974759/a1144fde-83b1-11e5-8039-d7f1d798561b.png)
